test/src/subnet/main.cpp

```diff
@@ -96,10 +96,10 @@ cadmium::dynamic::modeling::Ports oports_TOP = {typeid(outp_out)};
 cadmium::dynamic::modeling::Models submodels_TOP = {generator, subnet1};
 cadmium::dynamic::modeling::EICs eics_TOP = {};
 cadmium::dynamic::modeling::EOCs eocs_TOP = {
-  cadmium::dynamic::translate::make_EOC<Subnet_defs::out,outp_out>("subnet1")
+  cadmium::dynamic::translate::make_EOC<subnet_defs::out,outp_out>("subnet1")
 };
 cadmium::dynamic::modeling::ICs ics_TOP = {
-  cadmium::dynamic::translate::make_IC<iestream_input_defs<Message_t>::out,Subnet_defs::in>("generator","subnet1")
+  cadmium::dynamic::translate::make_IC<iestream_input_defs<message_t>::out,subnet_defs::in>("generator","subnet1")
```

Comment:
Many lines in this file are more than 80 characters in length (like this line)

Reply:
Changes done. Thank You.